### DISCUSSION TITLE
Feature Request: Click to copy the path

### DISCUSSION THREAD

**Discussion by jatinkrmalik** [ravimashru/obsidian-show-file-path#5] [Archive]
@ravimashru  Love the plugin, it would be nice if we could click (or right-click) to copy the path of the current file as you can then use it quickly to embed complex queries instead of having to manually type them in index pages.

```
# Index

## Daily Notes

query
path:"my/daily/notes/are/here/*"
```

**Comment by ravimashru** [comment]
Hello 👋  
Thanks for the input! 
This should be relatively simple to implement. I'll take a stab at it over the weekend. :)

**Comment by ravimashru** [comment]
@jatinkrmalik I just released version 0.3.0 that allows you to (left) click on the path to copy it to the clipboard.

The name of the file also gets copied based on whether you have enabled the `Show file name` setting.

Could you please see if this works for you?

**Reply by jatinkrmalik** [reply]
Works perfectly man! Thank you.
